api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>One idea to package this for a user is to create a new function that combines the above functions (including the get_data_points call) into one complete function so that there's only one entry point. </p><p> </p><p>For example, in order to interact with this a user would only just need to specify the five entity ids (region, metric, item, descendent level id, optional: source id, freq id)) they are interested in finding anomalies for. </p><p> </p><p>And then as a plus, if you wanted to turn this into a script, you can also make this runnable through a command line interface similar to quick_start.py. so a user can just find anomalies through anomaly_detect --region_id=21 --item_id=21 --metric_id=21 </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>